Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:9.0-alpine
+RUN npm install --global yarn create-react-app
+
+WORKDIR /usr/src/app
+COPY package.json /usr/src/app/
+# RUN yarn install
```

Comment:
Die Daten in der ES DB müssten hier wahrscheinlich erst noch initialisiert werden